include/dd/ComplexNumbers.hpp

```diff
@@ -181,6 +186,7 @@ namespace dd {
 
             return ret;
         }
+
```

Comment:
Any particular reason for that whitespace change here?

Reply:
I originally wrote the overload there, it's probably a leftover from when I copy-pasted. I removed it in another commit